services/rollout-service/pkg/cmd/server.go

```diff
@@ -157,6 +159,33 @@ func runServer(ctx context.Context, config Config) error {
 	broadcast := service.New()
 	shutdownCh := make(chan struct{})
 	ready := false
+	versionC := versions.New(overview)
```

Comment:
What happens if the overview service is down for a minute?
Do we then lose updates from Argo Cd?

Reply:
Discussed, it's true but we can live with this.
The rollout-service does on restart send everything necessary TO argoCd, though.

Reply:
Yes we do but we immediately fully recover once it's back.